tests/rbd/test_rbd_trash_purge_schedule.py

```diff
@@ -0,0 +1,152 @@
+import random
+import time
+
+from ceph.parallel import parallel
+from tests.rbd.rbd_utils import Rbd, initial_rbd_config
+from utility.log import Log
+from utility.utils import run_fio
+
+log = Log(__name__)
+
+
+def create_an_image_run_fio(rbd, pool, client, image_name):
+    if rbd.create_image(pool, image_name, "10G"):
+        log.error(f"Image creation failed for {image_name}")
+        return 1
+    run_fio(image_name=image_name, pool_name=pool, client_node=client, runtime=10)
+    return 0
+
+
+def create_images_run_fio_in_parallel(rbd, pool, client, num_images):
+    log.info(f"Creating {num_images} images in pool {pool} and initiating io")
+
+    with parallel() as p:
+        for i in range(0, num_images):
+            p.spawn(create_an_image_run_fio, rbd, pool, client, f"image_{i}")
+        return 0
+
+
+def move_all_images_to_trash(rbd, pool, num_images):
+    log.info("Moving all images in pool to trash")
+    for i in range(0, num_images):
```

Comment:
i think we can apply parallelism here also. as just moving image to trash.

Reply:
IMO, it is redundant to add it here as trash mv is instantaneous task
I used it in creation and io as it takes >30 seconds for each image